data_source_obmcs_core_console_history_data.go

```diff
@@ -36,7 +35,7 @@ func ConsoleHistoryDataDatasource() *schema.Resource {
 					}
 
 					if v < 10240 {
-						es = append(es, fmt.Errorf("expected %s to be in the range (%d - %d), got %d", k, min, max, v))
+						es = append(es, fmt.Errorf("expected %s to be less than %d, got %d", k, 10240, v))
```

Comment:
10240 could be put in a variable so that it doesn't have to be changed in multiple places